conans/model/options.py

```diff
@@ -29,10 +29,14 @@ def dumps(self, scope=None):
         if self._value is None:
             return None
         important = "!" if self.important else ""
+        value = self._value
+        if " " in value:
+            value = value.replace('"', r'\"')
+            value = f'"{value}"'
```

Comment:
I think this shouldn't be done in all cases while dumping the options, it could be breaking.
I was thinking of doing this just in the generation of ``build_args`` that happens in the ``build-order``.

Reply:
Yes, this is my same fear - it was just easier to test that this would work by doing it here, I should have marked this a draft to show that I'm just testing for now sorry :)